lib/src/generators/contract/openapi3.ts

```diff
@@ -64,7 +68,13 @@ export function openApiV3(contractDefinition: ContractDefinition): OpenApiV3 {
         operationId: endpoint.name,
         description: endpoint.description,
         tags: endpoint.tags,
-        parameters: getParameters(contractDefinition.types, endpoint),
+        parameters: getParameters(
+          contractDefinition.types,
+          endpoint,
+          contractDefinition.config
```

Comment:
In order to reduce the amount of conditional logic here, I'd suggest making `config` and `paramSerializationStrategy` required in the models. They could be set to a reasonable default value when parsing the TypeScript AST. Does that make sense?

Reply:
It makes sense, updated